AUTHORS

```diff
@@ -868,5 +867,128 @@ Super Zheng <superzheng@tencent.com>
+janriemer <janriemer@users.noreply.github.com>
+Arthur Gautier <superbaloo+registrations.github@superbaloo.net>
+Martin von Gagern <Martin.vGagern@gmx.net>
+Hideki Yamamura <hideki.yamamura@gmail.com>
+Lenny Markus <lmarkus@paypal.com>
+Nelson Pecora <nelson@yoshokatana.com>
+ronkorving <ron@ronkorving.nl>
+Xotic750 <xotic750@gmail>
+Nicholas Young <nicholas@nicholaswyoung.com>
+chrisjohn404 <chris.s.johnson4@gmail.com>
+Bo Borgerson <gigabo@gmail.com>
+Mudit Ameta <zeusdeux@gmail.com>
+Adri Van Houdt <adri.van.houdt@gmail.com>
+Max Barinov <me@maxbarinov.com>
+Damian Schenkelman <damian.schenkelman@gmail.com>
+iamchenxin <iamchenxin@gmail.com>
+Nik Nyby <nnyby@columbia.edu>
+Ben Ripkens <bripkens.dev@gmail.com>
+Ryan Sobol <contact@ryansobol.com>
+jona <jona@jona.io>
+Martin Bark <martin@barkynet.com>
+José F. Romaniello <jfromaniello@gmail.com>
+Richard Sun <rsun@media.ucla.edu>
+Sequoia McDowell <sequoia.mcdowell@gmail.com>
+zoubin <zoubin04@gmail.com>
+Robert Jefe Lindstaedt <robert.lindstaedt@gmail.com>
+Saquib <saquibofficial@gmail.com>
+Daniel Sellers <daniel@designfrontier.net>
+julien.waechter <julien.waechter@gmail.com>
+Vitor Cortez <vitoracortez+github@gmail.com>
+Michael Theriot <michael.lee.theriot@gmail.com>
+Peter Geiss <peter.w.geiss@gmail.com>
+Richard Lau <riclau@uk.ibm.com>
+José Moreira <jose.moreira@findhit.com>
+Tom Atkinson <atkinson.tommy@nsoft.ba>
+Glen Arrowsmith <glen.arrowsmith@gmail.com>
+Jeff Harris <techjeffharris@gmail.com>
+Alexander Makarenko <estliberitas@gmail.com>
+Drew Folta <drew@folta.net>
+Myles Borins <myles.borins@gmail.com>
+Jimb Esser <wasteland@gmail.com>
+Dmitriy Lazarev <wKich@users.noreply.github.com>
+Adam Langley <agl@google.com>
+Kári Tristan Helgason <kthelgason@gmail.com>
+Manuel Valls <manolo@vlrz.es>
+Prayag Verma <prayag.verma@gmail.com>
+Gibson Fahnestock <gib@uk.ibm.com>
+Alan Cohen <alan@ripple.com>
+Chris911 <christophe.naud.dulude@gmail.com>
+Matthias Bastian <dev@matthias-bastian.de>
+Phillip Kovalev <twilightfeel@gmail.com>
+Rainer Oviir <roviir@gmail.com>
+HUANG Wei <grubbyfans@gmail.com>
+dcposch@dcpos.ch <dcposch@dcpos.ch>
+Daniel Turing <mail@danielturing.com>
+Julie Pagano <julie.pagano@gmail.com>
+Ruben Bridgewater <ruben.bridgewater@fintura.de>
+Felix Becker <felix.b@outlook.com>
+Igor Klopov <igor@klopov.com>
+Tsarevich Dmitry <dimhotepus@users.noreply.github.com>
+Ojas Shirekar <ojas.shirekar@gmail.com>
+Noah Rose <noahroseledesma@seattleacademy.org>
+Rafael Cepeda <rcepeda1993@gmail.com>
+Chinedu Francis Nwafili <frankie.nwafili@gmail.com>
+Braydon Fuller <braydon@bitpay.com>
+Jiho Choi <jray319@gmail.com>
+Dirceu Pereira Tiegs <dirceutiegs@gmail.com>
+Marc Cuva <mjcuva@gmail.com>
+Chris Dew <cmsdew@gmail.com>
+Tristan Slominski <tristan.slominski@gmail.com>
+Hugo Wood <hx4.5@free.fr>
+Owen Smith <owen@omsmith.ca>
+Anton Andesen <a.andersen@readyforsky.com>
+Aayush Naik <aayushnaik17@gmail.com>
+nettofarah <nettofarah@gmail.com>
+firedfox <wangyang02@baidu.com>
```

Comment:
Could you please remove this line? My email address was changed to wangyang0123@gmail.com as line 974.